build/publish/PublishNupkgToBlobFeed.targets

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
```

Comment:
Revert the unnecessary whitespace too. The diff should show only the relevant change.